CURRENT_SUPPORTED_VERSIONS

```diff
@@ -1,5 +1,5 @@
+2.15
 2.14
 2.13
 2.12
 2.11
-2.10
```

Comment:
2.10 went EOL on 23 Sep 2025.
2.11 is an EUS release, and will be supported until some time in 2027.
2.12 maintenance ends on 09 Jun 2026.